sdk/core/azure-core-amqp/samples/internal/eventhub_get_properties_sample/eventhub_get_eventhub_properties_sample.cpp

```diff
@@ -119,9 +119,11 @@ EventHubPartitionProperties GetPartitionProperties(
       message);
 
   EventHubPartitionProperties properties;
+  bool error{false};
   if (result.Status == Azure::Core::Amqp::_internal::ManagementOperationStatus::Error)
   {
-    std::cerr << "Error: " << result.Message.ApplicationProperties["status-description"];
+    std::cerr << "Error: " << result.Description;
+    error = true;
```

Comment:
Returning error as a tuple doesn't seem very idiomatic to me. Should this be a throw?